src/main/java/uk/gov/mca/beacons/service/repository/BeaconPersonRepository.java

```diff
@@ -1,10 +1,18 @@
 package uk.gov.mca.beacons.service.repository;
 
+import java.util.List;
 import java.util.UUID;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 import uk.gov.mca.beacons.service.model.BeaconPerson;
 
 @Repository
 public interface BeaconPersonRepository
-  extends JpaRepository<BeaconPerson, UUID> {}
+  extends JpaRepository<BeaconPerson, UUID> {
+  @Query(
+    nativeQuery = true,
+    value = "SELECT * FROM beacon_person WHERE beacon_id = ?1"
+  )
+  List<BeaconPerson> findAllByBeaconId(UUID beaconId);
```

Comment:
Maybe not for this PR, but you could have in the `BeaconsService` not called through to the `BeaconsMapper` class and exposed two methods on here:

```java
@Query(
    nativeQuery = true,
    value = "SELECT * FROM beacon_person WHERE beacon_id = ?1 AND person_type = 'OWNER'"
  )
  BeaconPerson findOwnerByBeaconId(UUID beaconId);

@Query(
    nativeQuery = true,
    value = "SELECT * FROM beacon_person WHERE beacon_id = ?1 AND person_type = 'EMERGENCY_CONTACT'"
  )
  List<BeaconPerson> findAllEmergencyContactsByBeaconId(UUID beaconId);
```

and then manually assigned those to the `Beacon` object